src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -209,7 +211,7 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.CloudConfig == "" {
-		cfg.CloudConfig = "https://s3.amazonaws.com/lantern_config/cloud.2.0.0-nl.yaml.gz"
+		cfg.CloudConfig = fmt.Sprintf("https://s3.amazonaws.com/lantern_config/cloud.%v.yaml.gz", countryPlaceholder)
```

Comment:
Yeah that's a good point about it also changing in the yaml itself, which would make updating old clients annoying. OK let's leave it!